uint32 can_id
uint8 can_dlc
uint8[8] data
